README.md

```diff
@@ -11,14 +11,17 @@ Write a feature test in Gherkin and implement `steps_definitions` that best matc
 - Test the `Animal` object and ensure that it has the right output for its reader methods `name, type, age, old?`
 
 ## Instructions
+
 1. Checkout the sample feature & steps defs named `greeter.feature` & `greeter_steps.rb`
 2. `clone` your repo on to your computer
 3. `cd` into your directory
 4. `bundle` to pull in this project's dependencies
-5. navigate to `features/animal.feature` and write your feature
-6. navigate to `features/step_definitions/animal_steps.rb` and write your step defs
+5. `bundle exec cucumber` to run cucumber tests
+6. navigate to `features/animal.feature` and write your feature
+7. navigate to `features/step_definitions/animal_steps.rb` and write your step defs
 
 ## Resources
+
```

Comment:
Is there some reason we need this whitespace added here, and in the instructions section?  Looks fine to me as is.